docker/services/web/pytest/entrypoint.pytest.sh

```diff
@@ -3,8 +3,11 @@
 echo "sleeping entrypoint"
 sleep 120
 
+# python
+flask run --host=0.0.0.0
```

Comment:
to be honest, we don't need any command in this container at all, especially our web app
so this doesn't make sense. 
I suggest to remove entrypoint script at all 
and use `sleep infinity` instead

add the following line here `docker/docker-compose.pytest.yml` to the `web` service.
 
    command: sleep infinity

Reply:
Yes sleep infinity should work too.